tests/test_classifier/test_class_prediction_error.py

```diff
@@ -55,6 +55,8 @@ def test_integration_class_prediction_error_(self):
         model.fit(X, y)
         visualizer = ClassPredictionError(model, classes=["A", "B", "C", "D"])
         visualizer.score(X, y)
+        visualizer.finalize()
```

Comment:
I don't think the recent testsuite changes are necessarily impacting this yet outside of matplotlib 3.x entering in.

We have a PR almost ready to [contains a fix for this remove label issue](https://github.com/DistrictDataLabs/yellowbrick/pull/823/files#r291244069) that seems to have popped up with matplotlib 3.x.

Outside of that, I'd recommend adjusting the tolerances on these tests as the legend text will vary across platforms. 

The other option is removing the legend via `self.assert_images_similar(visualizer, remove_legend=True)

Reply:
Thanks, @ndanielsen! The `remove_legend` option is very helpful! 

As for the tolerances, @naresh-bachwani is creating these baseline images on Windows - we might want to create the baseline images on a *nix system just to keep things consistent. Perhaps when we get to the end of the PR I can generate the images and set the tolerances? This may only be required if the Travis tolerances are much higher than Windows.

Reply:
Great functionality @ndanielsen!

Reply:
@naresh-bachwani as promised, I've pushed a commit where the baseline images were generated on Unix -- waiting to see if this fixes the Travis tests and breaks the AppVeyor tests now.